app/views/moderations/actions_panel.html.erb

```diff
@@ -67,146 +50,51 @@
     </button>
     <span class="how-this-works-section">How does this work?</span>
   </div>
-
-  <% if current_user.has_role?(:super_admin) && @moderatable.class.name == "Article" %>
-    <h3> <a href="<%= @moderatable.path %>/edit">Edit Post</a> |
-      <a href="/internal/articles/<%= @moderatable.id %>" data-no-instant>Internal:Article</a> |
-      <a href="/admin/articles/<%= @moderatable.id %>" data-no-instant>Admin:Article</a></h3>
-  <% elsif current_user.has_role?(:super_admin) && @moderatable.class.name == "Comment" %>
-    <h3> <a href="/admin/comments/<%= @moderatable.id %>" data-no-instant>Admin:Comment</a> |
-      <a href="/admin/users/<%= @moderatable.user_id %>" data-no-instant>Admin:User</a></h3>
-  <% end %>
-  <p>
-    <b style="font-size:1.3em">All negative reactions are private.</b>
-  </p>
-  <p>
-    Use <b>thumbsdown</b> to move this content "down" for any reason (quality, usefulness, etc.).
-  </p>
-  <p>
-    Use <b>vomit</b> for code of conduct violations (harassment, being a jerk, spam, etc.).
-  </p>
-  <div class="other-things">
-    <header>
-      <h1 class="capitalized">Other things you can do</h1>
+  <%# Rating buttons and options go here, to be added by Arit %>
+  <div class="other-things-container">
+    <header class="other-things">
+      <h1>Other things you can do</h1>
     </header>
-  </div>
-  <div class="tag-mod-form">
-    <h2>Flag <a href="<%= @moderatable.user.path %>"><%= @moderatable.user.username %></a> as Abusive</h2>
-    <button class="reaction-button <%= Reaction.cached_any_reactions_for?(@moderatable.user, current_user, "vomit") ? "reacted" : "" %>"
-            data-reactable-id="<%= @moderatable.user.id %>"
-            data-category="vomit"
-            data-reactable-type="<%= @moderatable.user.class.name %>">
-      <%= image_tag("emoji/emoji-one-nausea-face-gray.png", alt: "Nausea face emoji") %>
-      <img class="reacted-emoji" src="<%= asset_path("emoji/emoji-one-nausea-face.png") %>" alt="Nausea down emoji" />
+    <button class="other-things-btn adjust-tags" type="button">
+      <div class="label-wrapper">
+        <div class="icon circle centered-icon">
+          <%= inline_svg_tag("mod-filled.svg", aria: true, title: "Adjust tags") %>
+        </div>
+        <header>
+          <h2>Adjust tags</h2>
+          <h3>Add or remove tags</h3>
+        </header>
+      </div>
+      <div class="toggle-chevron-container">
+        <%= inline_svg_tag("chevron-toggle.svg", aria: false) %>
+      </div>
     </button>
-    <p>
```

Comment:
A lot of removed code, since we technically don't need this in the actions panel file. If we need it for reference, we can get it from `app/views/moderations/mod.html.erb`.

Reply:
Absolutely! (the code was interfering with my process anyways lol)